File: server/data/results/translations/lite/plain/java/gcd_lcm.java
public class Writing {
    public static int gcd(int a, int b) {
        while (b != 0) {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }

    public static int lcm(int a, int b) {
        if (a == 0 || b == 0) {
            return 0;
        }
        return a / gcd(a, b) * b;
    }

    public static void main(String[] args) {
        int[][] pairs = {
            {12, 18},
            {7, 13},
            {100, 75}
        };
        for (int[] pair : pairs) {
            int a = pair[0];
            int b = pair[1];
            System.out.println(gcd(a, b) + " " + lcm(a, b));
        }
    }
}

File: server/data/results/translations/lite/plain/go/gcd_lcm.go
package main

import (
	"fmt"
)

func gcd(a, b int) int {
	for b != 0 {
		temp := b
		b = a % b
		a = temp
	}
	return a
}

func lcm(a, b int) int {
	if a == 0 || b == 0 {
		return 0
	}
	return (a / gcd(a, b)) * b
}

func main() {
	pairs := [][2]int{
		{12, 18},
		{7, 13},
		{100, 75},
	}
	for _, pair := range pairs {
		a, b := pair[0], pair[1]
		fmt.Println(fmt.Sprintf("%d %d", gcd(a, b), lcm(a, b)))
	}
}